okta-hosted-login/App.js

```diff
@@ -14,13 +14,13 @@ import React from 'react';
 
 import { Button, StyleSheet, Text, View } from 'react-native';
 import TokenClient from '@okta/okta-react-native';
-import config from './.samples.config';
+// import config from './.samples.config';
 
 const tokenClient = new TokenClient({
-  issuer: config.oidc.issuer,
-  client_id: config.oidc.clientId,
-  scope: config.oidc.scope,
-  redirect_uri: config.oidc.redirectUri
+  issuer: 'https://dev-403914.okta.com/oauth2/default',
```

Comment:
Remove all these hardcoded values in this file?

Reply:
removed hard coded values. The module will be deleted after this PR gets merged